src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImageFunctionalTest.groovy

```diff
@@ -43,6 +44,7 @@ class DockerBuildImageFunctionalTest extends AbstractGroovyDslFunctionalTest {
         result.output.contains("Created image with ID")
     }
 
+    @IgnoreIf({ os.windows })
```

Comment:
Why did this test case not work for you locally?

Reply:
I get: -
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':buildImage'.
Caused by: org.gradle.api.UncheckedIOException: Failed to create MD5 hash for file content.
Caused by: java.io.IOException: The process cannot access the file because another process has locked a portion of the file

The scan is here:-
https://gradle.com/s/2olp2ldq7davk

from stackoverflow: -
https://stackoverflow.com/questions/48632019/gradle-build-error-on-windows-failed-to-create-md5-hash-for-file/48894292

The issue here is that the destination directory contains the .gradle directory, which is used by the initial gradle process and is thus locked in Windows. Then the daemon tries to hash the lockfile, because it hashes all files in the destination dir to check if the task is up to date, but since it is a lockfile, it can't be accessed by any other process.

This test then cannot run in windows, but can in linux.

Reply:
Got it, thanks!